packages/docusaurus-theme-classic/src/theme/BlogPostPage/index.tsx

```diff
@@ -10,18 +10,22 @@ import React from 'react';
 import Layout from '@theme/Layout';
 import BlogPostItem from '@theme/BlogPostItem';
 import BlogPostPaginator from '@theme/BlogPostPaginator';
+import {DocTOC} from '@theme/DocItem';
 
 function BlogPostPage(props): JSX.Element {
   const {content: BlogPostContents} = props;
   const {frontMatter, metadata} = BlogPostContents;
   const {title, description, nextItem, prevItem, editUrl} = metadata;
+  const {hide_table_of_contents: hideTableOfContents} = frontMatter;
 
   return (
     <Layout title={title} description={description}>
       {BlogPostContents && (
-        <div className="container margin-vert--lg">
+        <div
+          className="container margin-vert--lg"
+          style={{maxWidth: 'inherit'}}>
```

Comment:
I'm sure if this was the correct way to handle the layout/style. 

I tried to use a 3-6-3 column layout, but because of the `max-width` property on the container it was very cramped. I didn't know whether it'd be best to override the `max-width` property, get rid of the `container` class name, or do something entirely different.

Reply:
not sure exactly what to do sorry 🤪  just know that adding the TOC shouldn't reduce as much the main content width

Reply:
Also avoid inline style objects, use real CSS instead

Reply:
@slorber For now, I made the blog TOC use `col--2` instead of `col--3` unlike the docs' TOC. Although it's narrower, it allows the content widths across all blog pages to be left untouched. I'll keep exploring other approaches and tinkering with the CSS